docs/dev/connectors/kafka.zh.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Apache Kafka Connector"
+title: "Apache Kafka 连接器"
```

Comment:
@wuchong  shall we translate proper nouns such as "connector"? Do we have a general translation with consensus or at least listed out in Glossary? IMHO there is no need to translate "connector".

Reply:
Yes. We have a consensus on connector.
https://cwiki.apache.org/confluence/display/FLINK/Flink+Translation+Specifications